php_1/db/PdoConnectionClass.php

```diff
@@ -0,0 +1,101 @@
+<?php
+class PdoConnectionClass
+{
+    private $server = "mysql:host=localhost;dbname=mydatabase3";
```

Comment:
define  the database connection parameters : host, user, password, database name as constants. Create a new class config.php and add these constants.